src/ArcxAnalyticsSdk.ts

```diff
@@ -75,6 +82,24 @@ export class ArcxAnalyticsSdk {
     )
   }
 
+  /*
+    Sent object in eth_sendTransaction is describe under link below:
+    https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_sendtransaction
+  */
+  private trackTransactions() {
```

Comment:
Add underscore because it's a private method
```suggestion
  private _trackTransactions() {
```